app/cells/modal/modal_cell.rb

```diff
@@ -0,0 +1,31 @@
+class ModalCell < UiComponents::Cell
+  attribute :id, mandatory: true, description: 'The HTML id attribute.'
+  attribute :content, mandatory: true, description: "The modal's content."
+  attribute :title, description: 'A title.'
+  attribute :buttons, description: 'An array of button types. ' \
+                                   'Valid types are `:close` and `:submit`.'
+
+  private
+
+  def buttons
+    @buttons.try(:map) do |button|
+      case button
+      when :close then close_button
+      when :submit then submit_button
+      else fail "'#{button}' button not implemented"
+      end
+    end.try(:join)
```

Comment:
Could we avoid these `try` calls here? Seems like we should bail/fail earlier if there are no buttons.